README.adoc

```diff
@@ -407,6 +407,19 @@ Retrieve the work item.
 $ ./bin/alm-cli show workitem --id 1 -H localhost:8080 --pp
 ----
 
+=== Reset Database
+
+The database are kept in a docker container that gets reused between restarts. Thus restarts will not clear out the database.
+
+To clear out the database kill the database like this:
```

Comment:
The first instructions says use `make dev` that starts the database via docker-compose.

I added this info since every time (since migrations) the db kept getting borked.